<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Rajitha" name="com.create4.execution.Ethic_add_TS_22" tests="8" failures="1" timestamp="09 Mar 2018 15:30:54 GMT" time="360.452" errors="1">
  <testcase name="CRe4_1080" time="39.379" classname="com.create4.execution.Ethic_add_TS_22"/>
  <testcase name="CRe4_1077" time="40.541" classname="com.create4.execution.Ethic_add_TS_22"/>
  <testcase name="CRe4_1079" time="40.736" classname="com.create4.execution.Ethic_add_TS_22"/>
  <testcase name="CRe4_1078" time="39.640" classname="com.create4.execution.Ethic_add_TS_22">
    <failure type="java.lang.AssertionError" message="expected [] but found [Title *]">
      <![CDATA[java.lang.AssertionError: expected [] but found [Title *]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.create4.scenarios.TS_22.CRe4_1078(TS_22.java:692)
	at com.create4.execution.Ethic_add_TS_22.CRe4_1078(Ethic_add_TS_22.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- CRe4_1078 -->
  <testcase name="CRe4_1070_71_1085_80_to_85" time="73.088" classname="com.create4.execution.Ethic_add_TS_22">
    <error type="org.openqa.selenium.NoSuchElementException" message="Cannot locate element with text:  
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1cd5757287168e54b817830adce9b0158d&#039;, time: &#039;2016-06-30 19:26:09&#039;
System info: host: &#039;Rajitha&#039;, ip: &#039;192.168.56.1&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_111&#039;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Cannot locate element with text:  
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'Rajitha', ip: '192.168.56.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_111'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.Select.selectByVisibleText(Select.java:150)
	at com.create4.scenarios.TS_22.CRe4_1070_71_1085_80_to_85(TS_22.java:474)
	at com.create4.execution.Ethic_add_TS_22.CRe4_1070_71_1085_80_to_85(Ethic_add_TS_22.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- CRe4_1070_71_1085_80_to_85 -->
  <testcase name="CRe4_1095_to_1100_1101_1102" time="41.255" classname="com.create4.execution.Ethic_add_TS_22"/>
  <testcase name="CRe4_1066_1067" time="45.219" classname="com.create4.execution.Ethic_add_TS_22"/>
  <testcase name="CRe4_1064_65" time="40.594" classname="com.create4.execution.Ethic_add_TS_22"/>
</testsuite> <!-- com.create4.execution.Ethic_add_TS_22 -->
